src/components/report-problem.component.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+import React, {
+  Component,
+  View,
+  StyleSheet,
+  TouchableHighlight,
+  Text
+} from 'react-native';
+import s from './report-problem.style';
+const styles = StyleSheet.create(s);
+
+var Camera = require('react-native-camera');
+
+var RNFS = require('react-native-fs');
```

Comment:
That's a library for reading and writing to the IOS device file system.  The camera stores the data in a file, and this reads it and sends it to firebase.  We need to review the performance though - streaming those images is slow.  I'm thinking the images need to be hosted on firebase and then something like an <image url="firebase.com/imagex"> used (not a base64 encoded string).